tardis/opacities/continuum/continuum_state.py

```diff
@@ -0,0 +1,162 @@
+import numpy as np
+
+
+class ContinuumState:
+    def __init__(
+        self,
+        nu_i,
+        level2continuum_idx,
+        p_fb_deactivation,
+        photo_ion_cross_sections,
+        chi_bf,
+        ff_cooling_factor,
+        fb_emission_cdf,
+        photo_ion_idx,
+        k_packet_idx,
+    ):
+        """Current State of the Continuum Required for Opacity Computation
+
+        Args:
```

Comment:
Wrong docstring style

Reply:
I think I used the wrong one for most of them, I'll go back and edit them